packages/labs/analyzer/src/test/browser/utils.ts

```diff
@@ -0,0 +1,215 @@
+import {Analyzer, type AbsolutePath} from '../../index.js';
+// import ts from '../../internal/typescript.js';
+import type ts from 'typescript';
+import * as path from 'path-module';
+
+/**
+ * Map of filenames -> versioned content
+ */
+interface Cache {
+  [index: string]: {content: string; version: number};
+}
+
+/**
+ * Map of filenames -> content
+ */
+export interface Files {
+  [index: string]: string;
+}
+
+// Note that because some paths come into the language host filesystem
+// abstractions in posix format and others may come in in OS-native
+// format, we normalize all paths going in/out of the in-memory cache to
+// posix format. There is apparently no path lib method to do this.
+// https://stackoverflow.com/questions/53799385/how-can-i-convert-a-windows-path-to-posix-path-using-node-path
+// const normalize = (p: string) => p.split(path.sep).join(path.posix.sep);
+const normalize = (p: string) => p;
```

Comment:
Remove this? I guess we don't have to worry about Window-style paths getting in in the browser?